tests/importers/test_multi_project.py

```diff
@@ -210,6 +210,32 @@ def test_not_importing_not_relevant_additional_files(tmpdir_factory):
     assert not selector.is_imported(str(not_relevant_file2))
 
 
+def test_not_importing_conversation_tests_in_project(tmpdir_factory):
+    root = tmpdir_factory.mktemp("Parent Bot")
+    config = {"imports": ["bots/Bot A"]}
+    config_path = str(root / "config.yml")
+    utils.dump_obj_as_yaml_to_file(config_path, config)
+
+    story_data_file = root / "bots" / "Bot A" / "data" / "stories.md"
+    story_data_file.write("""## story""", ensure=True)
+
+    conversation_tests_file = (
```

Comment:
to actually fully test the `is_conversation_test_file` function we need also to put a NLU file in the `tests` directory and make sure that that's actually loaded correctly.

Reply:
2nd thought: I think it's actually fine to omit this case here and rather add the test to the unit tests of `is_conversation_test_file`